World video exporter info (#454)
pupil-labs/pupil-docs: src/core/software/pupil-player.md

  <img src="../../media/core/imgs/export.jpg" style="display:flex;margin:0 auto;">
</div>

The export saves the world video as shown in Player, including all currently active visualizations (see  [Visualization Plugins](#visualization-plugins "Pupil Player visualization plugins documentation")). 

Gereral overview for exported video files:

> The World and Eye video exporter save the scene video file, together with [numpy](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html) and csv files containing timestamps corresponding to each frame. 

Reviewer: ```suggestion
The World and Eye video exporter saves the scene video file, together with [numpy](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html) and csv files containing timestamps corresponding to each frame. 
```

> The timestamps follow the [Pupil Time](/core/terminology/#timestamps) convention. The csv timestamp files include an additional `pts` column. `pts` is an abbreviation for _presentation timestamps_ and refers to the media file's internal time representation. It can be used to seek or identify specific frames within the media file. See this tutorial on how to extract individual frame images from the world video. 

Reviewer 2: TODO: add link to tutorial

Otherwise, I feel like this is a very good summary.

Reviewer 2: ```suggestion
The timestamps follow the [Pupil Time](/core/terminology/#timestamps) convention. The csv timestamp files include an additional `pts` column. `pts` is an abbreviation for _presentation timestamps_ and refers to the media file's internal time representation. It can be used to seek or identify specific frames within the media file. See this [tutorial on how to extract individual frame images from the world video](https://github.com/pupil-labs/pupil-tutorials/blob/master/09_frame_identification.ipynb). 
```

  <img src="../../media/core/imgs/export.jpg" style="display:flex;margin:0 auto;">
</div>

The export saves the world video as shown in Player, including all currently active visualizations (see the [Visualization Plugins](#visualization-plugins "Pupil Player visualization plugins documentation") section). 

Gereral overview for exported video files:

> The `world video exporter` exports the scene video file, together with numpy and .csv files containing timestamps corresponding to each frame. Data made available in world_timestamps.csv is timestamps in seconds from ..., and 'presentation timestamp' `pts`. This stands for presentation timestamp. It refers to the media file's internal time representation and can be used to seek to/identify specific frames within the media file.

Reviewer 2: ```suggestion
The World and Eye video exporter save the scene video file, together with [numpy](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html) and csv files containing timestamps corresponding to each frame. 
The timestamps follow the [Pupil Time](/core/terminology/#timestamps) convention. The csv timestamp files include an additional `pts` column. This stands for presentation timestamp and refers to the media file's internal time representation. It can be used to seek to/identify specific frames within the media file.
```

  <img src="../../media/core/imgs/export.jpg" style="display:flex;margin:0 auto;">
</div>

The export saves the world video as shown in Player, including all currently active visualizations (see  [Visualization Plugins](#visualization-plugins "Pupil Player visualization plugins documentation")). 

Gereral overview for exported video files:

The World and Eye video exporter save the scene video file, together with [numpy](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html) and csv files containing timestamps corresponding to each frame. 
The timestamps follow the [Pupil Time](/core/terminology/#timestamps) convention. The csv timestamp files include an additional `pts` column. `pts` is an abbreviation for _presentation timestamps_ and refers to the media file's internal time representation. It can be used to seek or identify specific frames within the media file.

Media time (seconds after start) can be calculated as follows: `pts / time_base = seconds since start`. 

> 

Author: I have edited down the text further @willpatera removing the last couple of sentences. @papr I also suggest adding link to the tutorial you made on this.
